pkg/sds/server/server_test.go

```diff
@@ -82,9 +79,7 @@ var _ = Describe("SDS Server", func() {
 							Type: "builtin",
 							DpCert: &mesh_proto.CertificateAuthorityBackend_DpCert{
 								Rotation: &mesh_proto.CertificateAuthorityBackend_DpCert_Rotation{
-									Expiration: &duration.Duration{
-										Seconds: 60,
-									},
+									Expiration: "1m",
```

Comment:
Can we have something more complex here, like "2m10s", just to verify this works too?

Reply:
sure